community/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/VarLengthExpandQueryPlanAcceptanceTest.scala

```diff
@@ -27,22 +27,23 @@ class VarLengthExpandQueryPlanAcceptanceTest extends ExecutionEngineFunSuite wit
     setUp("From")
     val query = "PROFILE MATCH (a:From {name:'Keanu Reeves'})-[*..4]->(e:To {name:'Andres'}) RETURN *"
     val result = executeWithCostPlannerOnly(query)
+
     result should havePlanLike(
       """
-        |+-----------------------+----------------+------+---------+------------------+--------------------------------------+
-        || Operator              | Estimated Rows | Rows | DB Hits | Variables        | Other                                |
-        |+-----------------------+----------------+------+---------+------------------+--------------------------------------+
-        || +ProduceResults       |              0 |    1 |       0 | a, e             | a, e                                 |
-        || |                     +----------------+------+---------+------------------+--------------------------------------+
-        || +Filter               |              0 |    1 |       5 | anon[*], a, e   | a:From AND a.name == {  AUTOSTRING0} |
-        || |                     +----------------+------+---------+------------------+--------------------------------------+
-        || +VarLengthExpand(All) |              0 |    4 |       8 | anon[*], a -- e | (e)<-[:*..4]-(a)                     |
-        || |                     +----------------+------+---------+------------------+--------------------------------------+
-        || +Filter               |              0 |    1 |       1 | e                | e.name == {  AUTOSTRING1}            |
-        || |                     +----------------+------+---------+------------------+--------------------------------------+
-        || +NodeByLabelScan      |              1 |    1 |       2 | e                | :To                                  |
-        |+-----------------------+----------------+------+---------+------------------+--------------------------------------+
-      """.stripMargin)
+        |+-----------------------+----------------+------+---------+------------------+-----------------------------------+
+        || Operator              | Estimated Rows | Rows | DB Hits | Variables        | Other                             |
+        |+-----------------------+----------------+------+---------+------------------+-----------------------------------+
+        || +ProduceResults       |              0 |    1 |       0 | anon[*], a, e   |                                   |
```

Comment:
This is an artefact of the test assertion removing the position of the anonymous variable before testing the output. The string is actually `anon[37]`, but to make the testing safe against newline differences between OSs, we remove the positions before comparing the plan output.